libs/vkd3d/breadcrumbs.c

```diff
@@ -599,21 +599,30 @@ void vkd3d_breadcrumb_tracer_report_device_lost(struct vkd3d_breadcrumb_tracer *
     if (device->vk_info.NV_device_diagnostic_checkpoints)
     {
         /* vkGetQueueCheckpointDataNV does not require us to synchronize access to the queue. */
-        queue_family_info = d3d12_device_get_vkd3d_queue_family(device, D3D12_COMMAND_LIST_TYPE_DIRECT);
+        queue_family_info = d3d12_device_get_vkd3d_queue_family(device,
+                                D3D12_COMMAND_LIST_TYPE_DIRECT,
```

Comment:
Just keep this on one line, the indents here are a bit goofy. Ditto the similar code below.